layout/formlayout.go

```diff
@@ -126,23 +120,19 @@ func (f *formLayout) Layout(objects []fyne.CanvasObject, size fyne.Size) {
 // For a FormLayout this is the width of the widest label and content items and the height is
 // the sum of all column children combined with padding between each.
 func (f *formLayout) MinSize(objects []fyne.CanvasObject) fyne.Size {
-	table := f.tableCellsSize(objects, 0)
+	labelWidth, contentWidth, heights := f.tableCellsSize(objects, 0)
+	if len(heights) == 0 {
```

Comment:
Nit: `len(heights)` is invoked 3 times in this func. Compiler is probably smart enough to inline it but it wouldn't hurt to cache it in a variable

Reply:
It is just a field on the slice struct. It basically has zero cost and the compiler should be able to optimise it. Coaching it might honestly be slower and prevent the compiler from optimising it.